external-crates/move/crates/move-vm-runtime/src/lib.rs

```diff
@@ -2,29 +2,38 @@
 // Copyright (c) The Move Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-#![forbid(unsafe_code)]
```

Comment:
would it make more sense to leave this and use `#[allow(unsafe_code)]` for the modules that needs it?
I think it may read better?

Reply:
I had to use `deny`, but it is done now.